ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: openshift-e2e-openstack-ipi
+  steps:
+    pre:
+    - chain: ipi-openstack-pre
+    post:
+    - chain: ipi-openstack-post
+    env:
```

Comment:
Needs to set `CLUSTER_TYPE`:
```
time="2020-09-16T08:22:56Z" level=error msg="Failed to get resolved config for test" error="failed resolve ReleaseBuildConfiguration: Failed resolve MultiStageTestConfiguration: e2e-openstack-ipi: openshift-e2e-openstack-ipi: ipi-openstack-pre: ipi-conf-openstack: ipi-conf-openstack: unresolved parameter: CLUSTER_TYPE" org=openshift repo=release 
```
I

Reply:
That is strange. Shouldn't  "CLUSTER_TYPE" be derived from the test definition's cluster_profile in https://github.com/openshift/release/pull/11000/files#diff-f9121dca038210d2d38e2713d851b4efR263

Reply:
Ho I see, CLUSTER_TYPE should NOT be listed under ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-ref.yaml